src/overlays/actors/ovl_En_Md/z_en_md.c

```diff
@@ -55,41 +55,66 @@ static ColliderCylinderInit sCylinderInit = {
+    /* 0x6 */ ENMD_ANIM_SEQ_SURPRISE_TO_ANNOYED,  // slightly raise arms from hips -> look away
+    /* 0x7 */ ENMD_ANIM_SEQ_SURPRISE_TO_AKIMBO,   // lower slightly raised arms to hips -> hands on hips
+    /* 0x8 */ ENMD_ANIM_SEQ_CURIOUS_TO_ANNOYED,   // tilted head and *slam* -> looking away
+    /* 0x9 */ ENMD_ANIM_SEQ_ANNOYED_TO_HALT,      // looking away -> halt gesture
+    /* 0xA */ ENMD_ANIM_SEQ_AKIMBO_TO_ANNOYED,    // hands on hips -> looking away
+    /* 0xB */ ENMD_ANIM_SEQ_STOP_WALKING          // stop walking -> akimbo
+} EnMdAnimSequence;
+
+typedef enum EnMdAnimIndex {
+    /*  0 */ ENMD_ANIM_INDEX_AKIMBO_LOOP_IDLE,   // hands on hips; default idle
+    /*  1 */ ENMD_ANIM_INDEX_AKIMBO_LOOP_UNUSED, // hands on hips; never set
+    /*  2 */ ENMD_ANIM_INDEX_AKIMBO_TO_HALT,     // hands on hips -> halt gesture
+    /*  3 */ ENMD_ANIM_INDEX_HALT_LOOP,          // halt gesture
+    /*  4 */ ENMD_ANIM_INDEX_HALT_TO_CURIOUS,    // halt gesture -> tilted head
+    /*  5 */ ENMD_ANIM_INDEX_CURIOUS_LOOP,       // tilted head
+    /*  6 */ ENMD_ANIM_INDEX_ANNOYED_LOOP,       // looking away
+    /*  7 */ ENMD_ANIM_INDEX_AKIMBO_TO_WALK,     // hands on hips -> walking
+    /*  8 */ ENMD_ANIM_INDEX_WALK_LOOP,          // walking
+    /*  9 */ ENMD_ANIM_INDEX_AKIMBO_TO_SURPISE,  // hands on hips -> slightly raised arms
+    /* 10 */ ENMD_ANIM_INDEX_AKIMBO_LOOP,        // hands on hips
+    /* 11 */ ENMD_ANIM_INDEX_CURIOUS_TO_ANNOYED, // tilted head -> looking away
+    /* 12 */ ENMD_ANIM_INDEX_ANNOYED_TO_HALT,    // looking away -> halt gesture
+    /* 13 */ ENMD_ANIM_INDEX_AKIMBO_TO_ANNOYED   // hands on hips -> looking away
+} EnMdAnimIndex;
 
 static AnimationInfo sAnimationInfo[] = {
-    { &gMidoHandsOnHipsIdleAnim, 0.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
-    { &gMidoHandsOnHipsIdleAnim, 0.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -10.0f },
-    { &gMidoRaiseHand1Anim, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
-    { &gMidoHaltAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
-    { &gMidoPutHandDownAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
-    { &gMidoAnnoyedPointedHeadIdle1Anim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
-    { &gMidoAnnoyedPointedHeadIdle2Anim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
-    { &gMidoAnim_92B0, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
-    { &gMidoWalkingAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
-    { &gMidoHandsOnHipsTransitionAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
-    { &gMidoHandsOnHipsIdleAnim, 0.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -8.0f },
-    { &gMidoSlamAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
-    { &gMidoRaiseHand2Anim, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
-    { &gMidoAngryHeadTurnAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
+    { &gMidoAkimboLoopAnim, 0.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { &gMidoAkimboLoopAnim, 0.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -10.0f },
+    { &gMidoAkimboToHaltAnim, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
```

Comment:
Im not sure "akimbo" is a very common term. I understand what it is now after looking it up. But I feel like the fact that I had to google it in the first place means theres probably a better word. But I guess "hands on hips" would get a bit long 😅

Reply:
hehe. yeah, that was the reason: because it was a tad long, i've changed it

probably just "idle" will do? it's a normal mood for Mido to stand like that anyway:
```
gMidoIdleLoop

/*  0 */ ENMD_ANIM_INDEX_IDLE_LOOP,        // hands on hips; default idle
/*  1 */ ENMD_ANIM_INDEX_IDLE_LOOP_UNUSED, // hands on hips; never set
```